ci/scripts/e2e-iceberg-sink-v2-test.sh

```diff
@@ -52,6 +53,8 @@ poetry run python main.py -t ./test_case/iceberg_source_all_delete.toml
 poetry run python main.py -t ./test_case/iceberg_source_explain_for_delete.toml
 poetry run python main.py -t ./test_case/iceberg_predicate_pushdown.toml
 
+echo "--- Running benchmarks"
+poetry run python main.py -t ./benches/*.toml
```

Comment:
I guess `main.py` doesn't support multiple input files now 🤔

Reply:
Can fix it later then. It's out of scope. Handle it together with test globbing.